index.bs

```diff
@@ -683,7 +683,7 @@ To <dfn>maybe process a navigation attribution source</dfn> given a <a spec="HTM
 1. If |browsingContext| is not a <a spec="html">top-level browsing context</a>, return.
 1. Let <var>attributionSource</var> be |navigationParams|'s [=navigation params/attribution source=].
 1. If |attributionSource| is null, return.
-1. If |attributionSource|'s [=attribution source/attribution destination=] is not [=same site=] to |navigationParams|'s
+1. If |attributionSource|'s [=attribution source/attribution destination=] does not equal |navigationParams|'s
```

Comment:
Not sure this is right, either. The attribution destination is a site, but it's being compared to an origin here.

Reply:
Yeah, probably better to get site from origin first and then compare the site. Will do this in a separate PR.